tss-esapi-sys/src/bindings/x86_64-unknown-linux-gnu.rs

```diff
@@ -3306,13 +3282,13 @@ impl Default for TPMS_AUTH_RESPONSE {
 }
 pub type TPMI_AES_KEY_BITS = TPM2_KEY_BITS;
 pub type TPMI_SM4_KEY_BITS = TPM2_KEY_BITS;
-pub type TPMI_CAMELLIA_KEY_BITS = TPM2_KEY_BITS;
+pub type TPMI_TPM2_CAMELLIA_KEY_BITS = TPM2_KEY_BITS;
```

Comment:
I know we are supporting version 2.3.3 and unfortunately we then have to deal with these kind of bugs. I wonder if it would be possible to bump the minimum supported tpm2-tss version soon.

Reply:
Hmm, yeah, we're going to tread carefully on that on the Parsec side - I'll try and think of a way of bumping the minimum supported version in the crate without burning down support for Parsec. 

Potentially keeping the 2.3.3 version supported in a separate branch and move on with `master`? (it wouldn't involve porting every new feature to it, only stuff that we in Parsec need, and important bugfixes)

Reply:
I don't think that forking the crate is going to be great for anyone.

Reply:
@puiterwijk - is there some standard way of making something like this work, that you know of? Or are we going to end up in hell either way we try

@Superhepper - what version did you have in mind? >=2.4.0 or straight up 3.0.0?

Reply:
Well, we could go back to the idea of having a feature for which version of the TSS you're using, and then change implementation details based on that (maybe just have both of them in a side-module).

Reply:
No, straight up 3.0 would not be advisable because I think there still a lot of people running 2.x. But some 2.4 release would be nice.

Reply:
(And then, based on the version during install, put in a fake version of the new constant, so that we can just assume the lateest version exists, but maybe it comes from bindgen, maybe from our own version)

Reply:
I've raised #207 - will start looking at it soon